toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/TableTopSceneView.kt

```diff
@@ -101,18 +101,25 @@ public fun TableTopSceneView(
     onDrawStatusChanged: ((DrawStatus) -> Unit)? = null,
     content: (@Composable TableTopSceneViewScope.() -> Unit)? = null
 ) {
-    Box(modifier = Modifier.fillMaxSize()) {
-        Icon(
-            imageVector = Icons.Default.Face,
-            contentDescription = null,
-            modifier = Modifier.fillMaxSize(),
-            tint = Color.Red
-        )
+    Box(modifier = modifier) {
+        tableTopSceneViewProxy.sceneViewProxy.setManualRenderingEnabled(true)
+
+        val localLifecycleOwner = LocalLifecycleOwner.current
```

Comment:
I suggest a shorter variable name:
```suggestion
        val lifecycleOwner = LocalLifecycleOwner.current
```